code/modules/effects/map_effects/map_helpers.dm

```diff
@@ -0,0 +1,31 @@
+/obj/effect/map_effect/map_helper
+	name = "some non-descript map helper (abstract type def)"
+	desc = ""
+	icon = 'icons/effects/map_effects.dmi'
+	layer = ABOVE_ALL_MOB_LAYER
+
+/obj/effect/map_effect/map_helper/Initialize(mapload)
+	..()
+	return INITIALIZE_HINT_QDEL
```

Comment:
If these markers shouldn't end up being used in production, they should crash on init if loaded or an unit test should look for them, so that they don't end up being placed and deployed

Reply:
There may be other mapping helper markers, like numbers or rulers or the like. They should not crash on init.

Reply:
If they are not meant to be placed on the map, they should either crash or an unit test should find them, so that the unit test will prevent them from being placed in away sites/maps/ruins or similar

If they are meant to be placed on the map, they shouldn't queue for deletion without doing anything else on initialization

Reply:
Ok so they are not just meant to be looked as references when mapping, but to be placed on the map itself too?
If that's the case, yes they don't need to block the UT